src/Illuminate/Testing/TestResponse.php

```diff
@@ -970,6 +971,58 @@ public function assertJsonMissingValidationErrors($keys = null, $responseKey = '
         return $this;
     }
 
+    /**
+     * Assert that the response contains the structure of the JsonResponse.
+     *
+     * @param  JsonResource  $resource
```

Comment:
We use FQN everywhere.

```suggestion
     * @param  \Illuminate\Http\Resources\Json\JsonResource  $resource
```

Reply:
Updated, thanks!